setup.py

```diff
@@ -2,12 +2,30 @@
 
 # Copyright (c) 2018 Red Hat, Inc.
 # All Rights Reserved.
+import re
+import subprocess
 
 from setuptools import setup, find_packages
 
 with open('README.md', 'r') as f:
     long_description = f.read()
 
+
+def version(default):
+    try:
+        process = subprocess.check_output(['git', 'describe', '--tags'], universal_newlines=True)
+        commit = re.match(r'([\d\.]+)[\d-]*([a-g0-9-]*)', process)
+        if commit:
+            if len(commit.group(2)) > 0:
+                return commit.group(2)[1:]
+            else:
+                return commit.group(1)
+        else:
+            return default
+    except (Exception):
+        return default
+
+
 setup(
     name="ansible-runner",
     version="2.0.0",
```

Comment:
The `version` method above doesn't seem to ever get called.

There's also the thorny issue of how to deal with official releases, which really would need to be "2.0.0".